content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -567,3 +567,122 @@ goods (i.e :guilabel:`Opening Stock` + purchases during the period - :guilabel:`
+         .. image:: india/tds-apply.png
+            :alt: TDS Apply
+
+   .. tab:: TCS
+
+      - Navigate to **Accounting --> Configuration --> Chart of Accounts**. open the desired account, and configure the TDS/TCS section.
+         .. image:: india/tcs-2061h-coa1.png
+            :alt: TCS 206(1H) configuration with CoA
+         .. image:: india/tcs-2061h-coa2.png
+            :alt: TCS 206(1H) configuration with CoA
+      - Based on the COA used during the recording of the Customer Invoice, the TCS threshold limit will be checked.
+         .. image:: india/tcs-coa-invoice-config.png
+            :alt: configuration CoA with Vendor Bill
+      - At the time of recording a Customer Invoice, if the limit specified in the TCS section configured in COA is exceeded, the system will advise you to apply TCS.
+         .. image:: india/tcs-alert.gif
+            :alt: TCS Alert
+      - The advice will disappear once you apply TCS.
+
+      **TCS Advise**
+
+      - The TCS advice will be visible on the Customer Invoice, indicating the applicable section under which TCS may be applied.
+         .. image:: india/tcs-warning.png
+            :alt: TCS Advise
+
+      **How to apply TCS?**
+
+      - TCS is directly applicable in the tax on invoice lines.
+
+.. note::
+   - In Odoo, the aggregate total is calculated for partners sharing the same PAN number, across all branches of the company.
+   - **Example:**
+      .. list-table::
+         :header-rows: 1
+         :widths: 10 20 10 20 15
+
+         * - **Branch**
+           - **Customer**
+           - **Invoice**
+           - **Transaction Amount (₹)**
+           - **PAN Number**
+         * - IN - MH
+           - XYZ Enterprise - GJ
+           - Invoice 1
+           - ₹50,000
+           - ABCPX1234E
+         * - IN - MH
+           - XYZ Enterprise - GJ
+           - Invoice 2
+           - ₹30,000
+           - ABCPX1234E
+         * - IN - MH
+           - XYZ Enterprise - MH
+           - Invoice 3
+           - ₹40,000
+           - ABCPX1234E
+         * - IN - DL
+           - XYZ Enterprise - GJ
+           - Invoice 4
+           - ₹20,000
+           - ABCPX1234E
+         * - IN - GJ
+           - XYZ Enterprise - MH
+           - Invoice 5
+           - ₹60,000
+           - ABCPX1234E
+
+      - **Aggregate Total** = 50,000 + 30,000 + 40,000 + 20,000 + 60,000 = ₹200,000
+      - The aggregate total for all customers (XYZ Enterprise - GJ, MH, DL) sharing the PAN number ABCPX1234E across all branches is ₹200,000.
+   - The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these limits, go to **Accounting --> Configuration --> Taxes** Advance options Tab and Sections.
+      .. image:: india/tds-tcs-section-modifiy.png
+         :alt: TDS/TCS section modify
```

Comment:
```suggestion
.. note::
   The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these
   limits, go to :menuselection:`Accounting --> Configuration --> Taxes`. In the :guilabel:`Advanced
   Options` tab, click on the  :icon:`fa-arrow-right` :guilabel:`(internal link)` icon of the
   :guilabel:`Section` field.
      .. image:: india/tds-tcs-section-modifiy.png
         :alt: TDS/TCS section modify
```
I also recommend renaming the image so that "modify" is spelled correctly.